definitions/marts/looker_studio_marts/ls_participant_profile_states.sqlx

```diff
@@ -12,59 +12,98 @@ config {
         id: "Unique ID of the record being pulled from the participant_profile_states table.",

```

Comment:
Same Question

Reply:
It is the same as the V2 file you approved previously.